.github/workflows/release.yml

```
@@ -1,39 +1,23 @@
-name: sonar-release
-# This workflow is triggered when publishing a new github release
+name: release
 on:
   release:
     types:
       - published
 
-env:
-  PYTHONUNBUFFERED: 1
-
 jobs:
-  sonar_release:
+  release:
     runs-on: ubuntu-latest
-    name: Start release process
     steps:
-      - name: Scanner release
-        id: scanner_release
-        with:
-          publish_to_binaries: true
-          attach_artifacts_to_github_release: false
-          run_rules_cov: false
-          slack_channel: scanner-guild
+      - name: Setup JFrog CLI
+        uses: jfrog/setup-jfrog-cli@v1
```

We've recently got a dependabot update to v2 that could make it to the general template. See https://github.com/SonarSource/sonar-dotnet/pull/4891
```suggestion
        uses: jfrog/setup-jfrog-cli@v2
```